### PR TITLE
Uds 1748b

### PR DESCRIPTION
**Pull request by scott-williams-az**
### Description

Fix broken images in unity-bootstrap-theme to prepare for Summery camp.

Using solution from https://github.com/ASU/asu-unity-stack/compare/dev...local-image-test
- unity-bootstrap-theme (I have removed all references of unsplash from the theme even if they were not identified in JIRA)
- component-carousel (updated Image caption for an intentionally broken image)

### Links

- [JIRA ticket](https://asudev.jira.com/browse/UDS-1748)
- [Unity reference site](https://asu.github.io/asu-unity-stack/)
- [Unity Design Kit](https://shared-assets.adobe.com/link/fb14b288-bf63-47e0-5d30-384de5560455)

### FOR APPROVERS

- [Percy build approval](https://percy.io/5eae92d9/-all-UDS-packages)

### CODE REVIEW

File: shared/assets/img/named/demo-phot-4.jpeg
Comment (mlsamuelson) [inline shared/assets/img/named/demo-phot-4.jpeg:1]: Let's replace this image. No unintentional QR codes!

**Review by mlsamuelson — CHANGES_REQUESTED** [review]

**Review by davidornelas11 — COMMENTED** [review]
Screen Shot 2024-07-15 at 1 30 32 PM
This was already an issue but should we change the Testimonial carousel images as well while we're fixing this?

File: shared/assets/img/named/demo-phot-4.jpeg
Comment (scott-williams-az) [reply shared/assets/img/named/demo-phot-4.jpeg:1]: Done

**Review by davidornelas11 — APPROVED** [review]
Looks good!

**Review by mlsamuelson — APPROVED** [review]

**Pull request merged**